Sources/Segment/Plugins/SegmentDestination.swift

```diff
@@ -124,6 +124,7 @@ public class SegmentDestination: DestinationPlugin {
                     switch result {
                         case .success(_):
                             storage.remove(file: url)
+                            self.cleanupUploads()
```

Comment:
Hey - why this line has been added here when there is another call of this method that happens independently from the result value?

https://github.com/segmentio/analytics-swift/pull/148/files#diff-92679f628b3def3ba245e4435edfcea09ea06a77277e6e2497aa331efbf46028R136

Reply:
Nice catch @grzegorzkrukowski !  The line below outside of the case needs to be nixed.